libcudacxx/docs/extended_api/synchronization_primitives/barrier.md

```diff
@@ -60,6 +61,43 @@ Programs shall ensure that this transformation would not introduce errors, for
 Under CUDA Compute Capability 6 (Pascal) or prior, an object of type
   `cuda::barrier` or `cuda::std::barrier` may not be used.
 
+## Shared memory barriers with transaction count
```

Comment:
This sounds like its currently implemented for all barriers in shared memory, like `__shared__ cuda::barrier<thread_scope_system> x;`, but that's not the case, right?

Reply:
True. It's the thread_scope_block specialization that has this capability.

Reply:
I now emphasize that it's the `cuda::barrier<thread_scope_block>` that has this capability (when it is located in shared memory).